src/main/java/org/openmrs/addonindex/backend/Bintray.java

```diff
@@ -133,4 +179,12 @@ private String packageUrlFor(AddOnToIndex addOnToIndex) {
 				details.getRepo(),
 				details.getPackageName());
 	}
+
+	private String downloadCountUrlFor(AddOnToIndex addOnToIndex) {
```

Comment:
It's nice to have this method (and you should incorporate it in the other newer code).